bitrise.yml

```diff
@@ -106,11 +106,6 @@ workflows:
           inputs:
             - command: global add @sentry/cli
           title: Install Sentry CLI
-      - fastlane@2.7:
```

Comment:
if you look in Fastfile, `refresh_dsyms` lane actually doesn't do anything

Reply:
our bitrise yml is also out of sync, so will need to get that caught up separately